localnet/kubernetes/values-gateway.yaml

```diff
@@ -0,0 +1,7 @@
+config:
+  query_node_rpc_url: tcp://validator-poktroll-validator:36657
```

Comment:
Do we intentionally have the word `validator` in there twice?

Reply:
@Olshansk yes. `validator` comes from the name of the release ([defined in tilt](https://github.com/pokt-network/poktroll/blob/main/Tiltfile#L102)) and `poktroll-validator` comes from the https://github.com/pokt-network/helm-charts/blob/main/charts/poktroll-validator/Chart.yaml#L2. Pretty standard practice, because that is how helm generates the boilerplate: https://github.com/pokt-network/helm-charts/blob/main/charts/poktroll-validator/templates/_helpers.tpl#L13